lib/pr_checker.js

```diff
@@ -172,6 +203,13 @@ class PRChecker {
     for (const [type, ci] of ciMap) {
       const name = CI_TYPES.get(type).name;
       logger.info(`Last ${name} CI on ${ci.date}: ${ci.link}`);
+      commits.forEach((c) => {
```

Comment:
So the logic here should be that we only log commits that came after the latest CI run. This code would log commits that came after each CI run type so it might show a lot of duplicates and just generally unhelpful data.